Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -303,21 +303,27 @@ public void ProcArgs_HostUrl_SonarcloudUrl_HostUrlAndSonarcloudUrlMissing()
         sut.IsValid.Should().BeTrue();
     }
 
-    [TestMethod]
-    public void ProcArgs_HostUrl_SonarcloudUrl_HostUrlIsDefaultSonarcloud()
+    [DataTestMethod]
+    [DataRow("https://sonarcloud.io")]
+    [DataRow("https://SonarCloud.io")]
+    [DataRow("https://SONARCLOUD.IO")]
+    [DataRow("https://sonarqube.us")]
+    [DataRow("https://SonarQube.us")]
+    [DataRow("https://SONARQUBE.US")]
+    public void ProcArgs_HostUrl_SonarCloudUrl_HostUrlIsAnySonarCloud(string hostUrl)
     {
-        var sut = CreateDefaultArgs(new ListPropertiesProvider([new Property(SonarProperties.HostUrl, "https://sonarcloud.io")]));
+        var sut = CreateDefaultArgs(new ListPropertiesProvider([new Property(SonarProperties.HostUrl, hostUrl)]));
 
         sut.ServerInfo.Should().NotBeNull();
         sut.ServerInfo.IsSonarCloud.Should().BeTrue();
-        sut.ServerInfo.ServerUrl.Should().Be("https://sonarcloud.io");
+        sut.ServerInfo.ServerUrl.Should().Be(hostUrl);
```

Comment:
Optional: Could use `Should().BeOfType` and `BeEquivalentTo` as in `PreArgProc_Region_EU`.

Reply:
I'd need to add expectedApiBaseUrl and expectedRegion, so I'll keep it as is